snippets/lvms-disconnected-ImageSetConfig.adoc

```diff
@@ -25,12 +25,12 @@ mirror:
   - name: registry.redhat.io/ubi9/ubi:latest <9>
   helm: {}
 ----
-<1> Add `archiveSize` to set the maximum size, in GiB, of each file within the image set.
-<2> Set the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values, unless you are using the Technology Preview OCI feature.
-<3> Set the registry URL for the storage backend.
-<4> Set the channel to retrieve the {product-title} images from.
-<5> Add `graph: true` to generate the OpenShift Update Service (OSUS) graph image to allow for an improved cluster update experience when using the web console. For more information, see _About the OpenShift Update Service_.
-<6> Set the Operator catalog to retrieve the {product-title} images from.
-<7> Specify only certain Operator packages to include in the image set. Remove this field to retrieve all packages in the catalog.
-<8> Specify only certain channels of the Operator packages to include in the image set. You must always include the default channel for the Operator package even if you do not use the bundles in that channel. You can find the default channel by running the following command: `oc mirror list operators --catalog=<catalog_name> --package=<package_name>`.
-<9> Specify any additional images to include in image set.
\ No newline at end of file
+<1> Set the the maximum size (in GiB) of each file within the image set.
+<2> Specify the back-end location in which you want to save the image set. This location can be a registry or a local directory. You must specify the `storageConfig` values unless you are using the Technology Preview OCI feature.
```

Comment:
Out-of-scope but I noticed back-end here, and wanted to cite the IBMSG because I think it should be back end, or something more specific as the guidelines suggest:
![image](https://github.com/openshift/openshift-docs/assets/77501157/50dfd43f-b348-4f01-ac1f-9b487c41f7e1)

Reply:
@jakobmoellerdev 
Is there a more specific term to replace "back end" in the following sentence:

"Specify the back end location in which you want to save the image set. This location can be a registry or a local directory. You must specify the `storageConfig` values unless you are using the Technology Preview OCI feature."

Reply:
How about for <2> we just remove back end?
For <3> Specify the storage URL for the ImageStream when using a registry. 

And then for 3 I would reference the existing ImageStream docs.

Reply:
@jakobmoellerdev 
Thank you for the info. Updated as per the suggestions.